docker/oidc-controller/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN dotnet publish -c Release -o /app/out --no-restore
 # Build runtime image
 FROM mcr.microsoft.com/dotnet/core/aspnet:3.1
 
-EXPOSE 5000
```

Comment:
Port should be left as `5000` as other services/configurations will depend on this when communicating on the internal Docker network.